src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -90,6 +93,11 @@ public void robotInit() {
         updateManager = new UpdateManager(
                 subsystems.drivebaseSubsystem);
         updateManager.startLoop(5.0e-3);
+
+        // Create and push Field2d to SmartDashboard.
+        field = new Field2d();
+        SmartDashboard.putData(field);
```

Comment:
This needs to move above the subsystem creation so it can be used from the drivebase subsystem